cfn-template/deeplearning.template

```diff
@@ -23,8 +23,9 @@
         "p3.2xlarge",
         "p3.8xlarge",
         "p3.16xlarge",
-        "g2.8xlarge",
-        "g2.2xlarge",
```

Comment:
C5 instances, AMD Instances ?

Reply:
Updated C5, AMD instance has not been supported in DLAMI yet.